### ISSUE TITLE
Icons missing after upgrading to 3.0.11 

### ISSUE THREAD

**Issue by darthf1** [coreui/coreui-vue#100]
After upgrading from `3.0.10` to `3.0.11` my icons in the sidebar on the left disappear. I don't see anything in the changelog/releaselog. Does the update need additional config? I wouldn't expect anything to break, this being a patch release.

**Comment by woothu** [comment]
Have you updated also @coreui/icons or other packages?

**Comment by darthf1** [comment]
I dont have `@coreui/icons` in my `package.json`, it's already included in this package right? My `yarn.lock`:
```
"@coreui/icons-vue@^1.3.3":
  version "1.3.3"
  resolved "https://registry.yarnpkg.com/@coreui/icons-vue/-/icons-vue-1.3.3.tgz#7dd3f2dab0a681834f6be42cb00a5b1625bab88a"
  integrity sha512-TAhm1w5/ZAWyVoiRdpAFvLsYbRbf8pNnHT+0yordHehJLW+zbnwk00AHj694E+p0i0GjaZ7cOLVugOnOCGBn/A==
  dependencies:
    vue "~2.6.11"
```

I just did a full clean install like you said, and it's still happening. Reverting to `3.0.10` immediately fixes the issue.

**Comment by woothu** [comment]
Thank you for this report. There was a problem with @coreui/icons-vue package. Problem fixed with 3.0.12 version.
